src/org/opensolaris/opengrok/web/WebappListener.java

```diff
@@ -50,11 +48,6 @@
         implements ServletContextListener, ServletRequestListener {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(WebappListener.class);
-    private static final String ENABLE_AUTHORIZATION_WATCH_DOG = "enableAuthorizationWatchDog";
```

Comment:
where did this go?

Reply:
It was not used at all.

Reply:
but it should have been, when did this disappear?

Reply:
Oh I see, the code is still there, but the thread and the constants were moved to different file. Ok